.github/workflows/codeql.yml

```diff
@@ -1,50 +1,64 @@
-name: "Code scanning - action"
+name: "Code Scanning - Action"
 
 on:
   push:
+    branches:
+      - main
+      - release-*
   pull_request:
+    branches:
+      - main
+      - release-*
   schedule:
-    - cron: '0 19 * * 0'
+    #        ┌───────────── minute (0 - 59)
+    #        │  ┌───────────── hour (0 - 23)
+    #        │  │ ┌───────────── day of the month (1 - 31)
+    #        │  │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #        │  │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #        │  │ │ │ │
+    #        │  │ │ │ │
+    #        │  │ │ │ │
+    #        *  * * * *
+    - cron: '30 1 * * 0'
```

Comment:
Love this.

Reply:
Still loving it, no matter who added it. I **always** have to look up the Cron syntax.

Reply:
I just use https://crontab.guru/

Reply:
Why does the minute come first and why are hours 0-based but dates are 1-based?  That's confusing as hell.